src/data/markdown/docs/30 xk6-browser/01 xk6-browser/01 Browser/contexts.md

```diff
@@ -0,0 +1,25 @@
+---
+title: 'contexts()'
+excerpt: 'xk6-browser: Browser.contexts method'
+---
+
+Allows you to access all open [BrowserContext](/javascript-api/xk6-browser/browsercontext/)s.
+
+### Returns
+
+| Type  | Description                                                                    |
+| ----- | ------------------------------------------------------------------------------ |
+| Array | Array of [BrowserContext](/javascript-api/xk6-browser/browsercontext/) objects |
+
+
+## Example
+
+<!-- eslint-skip -->
+
+```javascript
+const browser = chromium.launch();
+console.log(browser.contexts().length); // prints `0`
+
+const context = browser.newContext();
+console.log(browser.contexts().length); // prints `1`
```

Comment:
It might be more useful to print the actual BrowserContext object rather than the length.

Reply:
Not sure how useful it is, but i've added it to commit 3d99d2381d596c0bdb2ff55d937e26ba41f1ac91